### PR TITLE
Adding the ability to toggle ARRAffinity in PS

### PR DESCRIPTION
**Pull request by jocawtho**
## Description

This adds a flag to allow customers to turn off/on ARRAffinity for their webapps. I also added this to the test and help files.

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by MiYanni** [comment]
@jocawtho It looks like:
`TestSetWebApp: Cannot convert 'System.String' to the type 'System.Boolean' required by parameter 'ClientAffinityEnabled'.`

If you wrap the parameter in parenthesis, it should resolve just fine. you should have ran into this locally. Also, you need to re-record the test since you changed it.

**Comment by jocawtho** [comment]
Thanks for this tips! Sorry, just learned how to run these locally. New to PS development. Fixes have been committed and pushed

File: src/ResourceManager/Websites/Commands.Websites/Cmdlets/WebApps/SetAzureWebApp.cs [hunk 1]
```diff
@@ -151,7 +154,8 @@ public override void ExecuteCmdlet()
                             Location = location,
                             ServerFarmId = WebApp.ServerFarmId,
                             Identity = parameters.Contains("AssignIdentity") && AssignIdentity ? new ManagedServiceIdentity("SystemAssigned", null, null) : WebApp.Identity,
-                            HttpsOnly = parameters.Contains("HttpsOnly") ? HttpsOnly : WebApp.HttpsOnly
+                            HttpsOnly = parameters.Contains("HttpsOnly") ? HttpsOnly : WebApp.HttpsOnly,
+                            ClientAffinityEnabled = parameters.Contains("ClientAffinityEnabled") ? ClientAffinityEnabled : WebApp.ClientAffinityEnabled
```
Comment (cormacpayne) [inline src/ResourceManager/Websites/Commands.Websites/Cmdlets/WebApps/SetAzureWebApp.cs]: @jocawtho in the scenario test above, you get the value for `ClientAffinityEnabled` from a web app, but if you hadn't provided a value at all, this line would still get the value from the web app. Is there a scenario where the user would provide the value for `ClientAffinityEnabled` from anywhere but a web app? If not, then we should remove the parameter and just set the value to `WebApp.ClientAffinityEnabled` all the time

File: src/ResourceManager/Websites/ChangeLog.md [hunk 2]
```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Current Release
+* Added -AssignIdentity & -Httpsonly properties for Set-AzureRmWebApp
```
Comment (cormacpayne) [inline src/ResourceManager/Websites/ChangeLog.md]: @jocawtho this change log entry needs to be updated -- it looks like it was copied from one of the previous entries

File: src/ResourceManager/Websites/Commands.Websites/Cmdlets/WebApps/SetAzureWebApp.cs (hunk 1 above)
Comment (jocawtho) [reply src/ResourceManager/Websites/Commands.Websites/Cmdlets/WebApps/SetAzureWebApp.cs]: Customer's should be able to set this setting without getting the setting from a webApp, but I chose to get the bool value from a webApp property to ensure the test's accuracy by setting the opposite value no matter what the default may be.

**Comment by panchagnula** [comment]
@jocawtho we have an item to update the set commadlet  for apps & slots to use the  'switch parameter' instead of the bool property. See #6066, we should all new parameters to follow this. So please go -ahead & update your code to use the switch parameter, if you cannot update the rest of the properties i will do so for the rest.

**Comment by maddieclayton** [comment]
Closing as comments have not been addressed in over a week. Please reopen once you are ready to address comments.

**Pull request closed without merging**